frontend/lib/immediates/num.h

```diff
@@ -76,8 +76,7 @@ enum IF1_string_kind : uint8_t {
 #define BOOL_SYS_WIDTH 0
 
 enum IF1_bool_type : uint8_t {
-  BOOL_SIZE_SYS, BOOL_SIZE_8, BOOL_SIZE_16, BOOL_SIZE_32,
-  BOOL_SIZE_64, BOOL_SIZE_NUM
+  BOOL_SIZE_SYS
```

Comment:
Is it not worth the effort to remove `BOOL_SIZE_SYS` even though it's now an effectively meaningless sentinel?

Reply:
That's a good question.  I balked at removing it completely because we support `bool` `Immediate`s that have a `num_index` field that stores it (or, for integers, reals, etc. stores the size of the integer, etc.).  I felt a bit nervous and pressed for time to try and see what would happen if we removed it from those cases (and, generally, am not very familiar with the current state of Immediates) so left it alone out of fear and trepidation.  I believe that Immediates are the main (only?) think that still use the value, though.